pkg/config/admin_secrets_test.go

```diff
@@ -14,15 +15,17 @@ func TestAdminSecrets(t *testing.T) {
 
 	testFile, err := ioutil.ReadFile("admin_secrets_test.yaml")
 	assert.NoError(t, err)
+	secretsFromFile := config.AdminSecretsConfig{}
+	err = yaml.Unmarshal(testFile, &secretsFromFile)
 
 	service := config.AdminSecretsService{}
 	service.Flags.DefaultAdminPassword = "mysecret"
 	err = service.NewAdminSecretsConfig()
 	assert.NoError(t, err)
 
-	s, err := service.Secrets.String()
-	tests.Debugf("%s", s)
+	secretsFromService := service.Secrets
+	tests.Debugf("%s", secretsFromService)
 	assert.NoError(t, err)
 
-	assert.Equal(t, string(testFile), s, "expected admin secret values do not match")
+	assert.Equal(t, secretsFromFile, secretsFromService, "expected admin secret values do not match")
```

Comment:
assuming `secretsFromFile` has an `Equals`y type thing?

Reply:
I changed one character in the text file and the test failed, so that was good enough for me.